src/components/[guild]/AccessHub/AccessHub.tsx

```diff
@@ -31,128 +15,47 @@ const DynamicCreatedPageCard = dynamic(() =>
   import("./components/CreatePageCard").then((m) => m.CreatePageCard)
 )
 
-export const useAccessedGuildPlatforms = (groupId?: number) => {
-  const { guildPlatforms, roles } = useGuild()
-  const { isAdmin } = useGuildPermission()
-  const { roleIds } = useMembership()
-
-  const relevantRoles = groupId
-    ? roles.filter((role) => role.groupId === groupId)
-    : roles.filter((role) => !role.groupId)
-
-  const relevantGuildPlatformIds = relevantRoles.flatMap((role) =>
-    role.rolePlatforms.map((rp) => rp.guildPlatformId)
-  )
-  const relevantGuildPlatforms = guildPlatforms.filter(
-    (gp) =>
-      relevantGuildPlatformIds.includes(gp.id) &&
-      gp.platformId !== PlatformType.POINTS &&
-      gp.platformId !== PlatformType.ERC20
-  )
-
-  // Displaying CONTRACT_CALL rewards for everyone, even for users who aren't members
-  const contractCallGuildPlatforms =
-    relevantGuildPlatforms?.filter(
-      (guildPlatform) => guildPlatform.platformId === PlatformType.CONTRACT_CALL
-    ) ?? []
-
-  if (isAdmin) return relevantGuildPlatforms
-
-  if (!roleIds) return contractCallGuildPlatforms
-
-  const accessedRoles = roles.filter((role) => roleIds.includes(role.id))
-  const accessedRolePlatforms = accessedRoles
-    .map((role) => role.rolePlatforms)
-    .flat()
-    .filter((rolePlatform) => !!rolePlatform)
-  const accessedGuildPlatformIds = [
-    ...new Set(
-      accessedRolePlatforms.map((rolePlatform) => rolePlatform.guildPlatformId)
-    ),
-  ]
-  const accessedGuildPlatforms = relevantGuildPlatforms?.filter(
-    (guildPlatform) =>
-      accessedGuildPlatformIds.includes(guildPlatform.id) ||
-      guildPlatform.platformId === PlatformType.CONTRACT_CALL
-  )
-
-  return accessedGuildPlatforms
-}
-
 const AccessHub = (): JSX.Element => {
   const { featureFlags, guildPin, groups, roles } = useGuild()
 
   const group = useRoleGroup()
   const { isAdmin } = useGuildPermission()
   const { isMember } = useMembership()
 
-  const accessedGuildPlatforms = useAccessedGuildPlatforms(group?.id)
-  const accessedGuildPoints = useAccessedGuildPoints("ACCESSED_ONLY")
-  const accessedGuildTokens = useTokenRewards(!isAdmin)
-
   const shouldShowGuildPin =
     !group &&
-    featureFlags.includes("GUILD_CREDENTIAL") &&
+    featureFlags?.includes("GUILD_CREDENTIAL") &&
     ((isMember && guildPin?.isActive) || isAdmin)
 
   const hasVisiblePages = !!groups?.length && roles?.some((role) => !!role.groupId)
 
-  const showAccessHub =
-    isAdmin ||
-    isMember ||
-    !!accessedGuildPlatforms?.length ||
-    (hasVisiblePages && !group)
+  const showAccessHub = isAdmin || isMember || (hasVisiblePages && !group)
```

Comment:
Shouldn't we remove the isMember condition? We should show it if there are visible pages now, or for the admin to set up pages / pin

Reply:
You're right, and we should actually add the `shouldShowGuildPin` boolean, fixed in https://github.com/guildxyz/guild.xyz/pull/1436/commits/a00fb20d4d7cf41fde49cea195a3ccff37d4e766!